sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb

```diff
@@ -0,0 +1,420 @@
+    "with open(train_annotations_file, \"w\") as train_f:\n",
+    "    with open(validation_annotations_file, \"w\") as validation_f:\n",
+    "        for className in os.listdir(src_images):\n",
+    "            subDir = src_images + className\n",
+    "            if not os.path.isdir(subDir):\n",
+    "                continue\n",
+    "            # Scan each sub directary\n",
+    "            print(\"Parsing \" + subDir)\n",
+    "            for image in os.listdir(subDir):\n",
+    "                json_line = dict(json_line_sample)\n",
+    "                json_line[\"image_url\"] += f\"{className}/{image}\"\n",
+    "                json_line[\"label\"] = className\n",
+    "\n",
+    "                if index % train_validation_ratio == 0:\n",
+    "                    # validation annotation\n",
+    "                    validation_f.write(json.dumps(json_line) + \"\\n\")\n",
+    "                else:\n",
+    "                    # train annotation\n",
+    "                    train_f.write(json.dumps(json_line) + \"\\n\")\n",
+    "                index += 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Basic pipeline job with Image Classification Multiclass task\n",
+    "\n",
+    "## 2.1 Build pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note that the used docker image doesn't suit for all size of gpu compute. Please use the following command to create gpu compute if experiment failed\n",
+    "# !az ml compute create -n gpu-cluster --type amlcompute --min-instances 0 --max-instances 4 --size Standard_NC12"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@pipeline(\n",
+    "    description=\"AutoML Image Clasiification Multiclass Pipeline\",\n",
+    "    default_compute=\"gpu-cluster\",\n",
+    ")\n",
+    "def automl_image_classification(\n",
+    "    image_classification_train_data,\n",
+    "    image_classification_validation_data\n",
+    "):\n",
+    "    # define the automl image_classification task with automl function\n",
+    "    image_classification_node = image_classification(\n",
+    "        training_data=image_classification_train_data,\n",
+    "        validation_data=image_classification_validation_data,\n",
+    "        target_column_name=\"label\",\n",
+    "        primary_metric=\"Accuracy\",\n",
+    "        properties={\"_automl_internal_label\": \"latest\",},\n",
+    "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes \n",
+    "        outputs={\"best_model\": Output(type=\"mlflow_model\")},\n",
+    "    )\n",
+    "    \n",
+    "    image_classification_node.set_limits(timeout_minutes=180)\n",
+    "    \n",
+    "    image_classification_node.extend_search_space(\n",
```

Comment:
Can we use the same search space as the one used in cli examples and standalone examples.